tools/benchmark_tool/README.md

```diff
@@ -117,138 +142,157 @@ Options:
+Running the application with the empty list of options yields the usage message given above and an error message.
 
-> **NOTE**: Before running the tool with a trained model, make sure the model is converted to the OpenVINO format (\*.xml + \*.bin) using the [Model Optimizer tool](../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
+### More information on inputs
+The benchmark tool supports topologies with one or more inputs. If a topology is not data sensitive, you can skip the input parameter, and the inputs will be filled with random values. If a model has only image input(s), provide a folder with images or a path to an image as input. If a model has some specific input(s) (besides images), please prepare a binary file(s) that is filled with data of appropriate precision and provide a path to it as input. If a model has mixed input types, the input folder should contain all required files. Image inputs are filled with image files one by one. Binary inputs are filled with binary inputs one by one.
 
 ## Examples of Running the Tool
+This section provides step-by-step instructions on how to run the Benchmark Tool with the `googlenet-v1` public model on CPU or GPU devices.  The [dog.bmp](https://storage.openvinotoolkit.org/data/test_data/images/224x224/dog.bmp) file is used as an input.
 
-This section provides step-by-step instructions on how to run the Benchmark Tool with the `googlenet-v1` public model on CPU or GPU devices. The [dog.bmp](https://storage.openvinotoolkit.org/data/test_data/images/224x224/dog.bmp) file is used as an input.
-
-> **NOTE**: The Internet access is required to execute the following steps successfully. If you have access to the Internet through the proxy server only, please make sure that it is configured in your OS environment.
+> **NOTE**: Internet access is required to execute the following steps successfully. If you have access to the Internet through a proxy server only, please make sure that it is configured in your OS environment.
 
 1. Install OpenVINO Development Tools to work with Caffe* models:
 
    ``` sh
    pip install openvino-dev[caffe]
    ```
 
-2. Download the model. Go to the Model Downloader directory and run the `omz_downloader` tool with the model name and directory to download the model to:
+2. Download the model. Go to the Model Downloader directory and run the `omz_downloader` script with specifying the model name and directory to download the model to:
 
    ```sh
    omz_downloader --name googlenet-v1 -o <models_dir>
    ```
-3. Convert the model to the OpenVINO IR format. Run Model Optimizer with the path to the model, model format and output directory to generate the IR files:
+
+3. Convert the model to the OpenVINO IR format. Run the Model Optimizer using the `mo` command with the path to the model, model format and output directory to generate the IR files:
```

Comment:
Do you think we should remove the entire point 3?

Reply:
Not for now - it is required for this model. But we should simplify this example to use a model that does not need to be converted, or caffe to be installed.  

For example:
1. `pip install openvino-dev` 
2. `omz_downloader --name asl-recognition-0004 --precisions FP16 --output_dir omz_models`
3. `benchmark_app -m omz_models/intel/asl-recognition-0004/FP16/asl-recognition-0004.xml -d CPU -hint throughput -progress`

@EdjeElectronics what do you think? This does not include the dog.bmp picture, but that also makes the example more generic.

The current example works, and for this PR we can definitely leave it as is, but it should not be required to install caffe and convert a model just to run a benchmark_app sample.

Reply:
Hi @helena-intel, I agree that the example should be simplified. I had left the example section untouched since it was already mostly clear and understandable, but it can definitely be improved. I'll test out the steps you listed and then add them to both the C++ and Python pages.